frontend/src/__tests__/integration/pages/projects/ProjectView.spec.ts

```diff
@@ -26,6 +24,10 @@ test('Project view page', async ({ page }) => {
   await expect(page.getByText('Non-DS Project 1', { exact: true })).toBeVisible();
   await expect(page.getByText('Non-DS Project 2', { exact: true })).toBeVisible();
   await expect(page.getByText('Non-DS Project 3', { exact: true })).toBeVisible();
+
+  await page.locator('#search-field-toggle').click();
+  await expect(page.getByRole('listitem').first()).toHaveText('Name');
+  await expect(page.getByRole('listitem').last()).toHaveText('User');
```

Comment:
Yay Tests! didn't even consider that. Can we also make a test to say has only 2 options? Seems weird to check only first/last... there could be one in the middle.